_posts/2020-08-09-actual-advice-1.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: Actual Advice #1: Start at a suistainable pace
+---
+
+
+This is the first blog post of a format I'd like to use to share advice I received throughout my career. There is so much advice out there that in most cases, you'll always find someone defending and someone else challenging any type of advice, so I won't claim this advice to be universally correct. I'd like to share the advice that worked for me personally, that made me better, not necessarily only at my job.
+
+I received this first advice from my highly valued colleague and co-worker Daniel Marbach. While he is a genius software engineer, he is also a gym enthusiast. Some day, I expressed my frustration about the lack of enjoyment and motivation I get from doing fitness exercises. I've mostly tried bodyweight training as I could do my exercises at home but I failed at completing exercise goals several times. I struggled to find motivation, causing me to find excuses to skip training again and again. At some point, you just stop completely because you give up your hope of achieving your training plan. Daniel's advise was very simple and you probably wonder why I even bother to write this down: Set yourself a goal that you are confident to achieve every single time. For me, this goal was to do my exercises just once a week.
```

Comment:
```suggestion
I received this first advice from my highly valued colleague and co-worker, Daniel Marbach. While he is a genius software engineer, he is also a gym enthusiast. Some day, I expressed my frustration about the lack of enjoyment and motivation I get from doing fitness exercises. I've mostly tried bodyweight training as I could do my workouts at home, but I failed at completing exercise goals several times. I struggled to find motivation, causing me to find excuses to skip training again and again. At some point, you stop because you give up your hope of achieving your training plan. Daniel's advice was straightforward, and you probably wonder why I even bother to write this down: Set yourself a goal that you are confident of achieving every single time. For me, this goal was to do my exercises just once a week.
```